app.py

```diff
@@ -94,8 +108,10 @@ def fetchStatus(data):
     remoteLogMsg = {"message":msg, "showTimePrefix":True}
     print(msg)
     emit('remoteLogMsg', remoteLogMsg)
-    emit('updateSensorData', sensordata, broadcast=True)
-    sendmessage(weather_message, True, True)
+    if enableSense or enablePimeteo:
+        emit('updateSensorData', sensordata, broadcast=True)
+    if enableWeather:
+        sendmessage(weather_message, True, True)
```

Comment:
These checks were needed cause if you disable weather, then in my cause it wanted to still send message, but the `weather_message` string is empty.

And this would be the same for the `sensordata`